### PR TITLE
feat: Set limit for advanced image processing images

### PR DESCRIPTION
**Pull request by cecheta**
Required for #750

## Purpose

* Set a limit to the number of images that are passed to the LLM when using advanced image processing. Defaults to 1

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## How to Test
* Deploy code, with advanced image processing enabled
* Upload similar images
* Make query about images
* Check number of images passed to LLM

### CODE REVIEW

**Review by adamdougal — APPROVED** [review]
LGTM! I assume search returns images in order of closest match?

**Review by ross-p-smith — APPROVED** [review]

File: code/backend/batch/utilities/helpers/env_helper.py [hunk 1]
```diff
@@ -114,6 +114,9 @@ def __load_config(self, **kwargs) -> None:
         self.USE_ADVANCED_IMAGE_PROCESSING = self.get_env_var_bool(
             "USE_ADVANCED_IMAGE_PROCESSING", "False"
         )
+        self.ADVANCED_IMAGE_PROCESSING_MAX_IMAGES = int(
```
Comment (frtibble) [inline code/backend/batch/utilities/helpers/env_helper.py:117]: nit: I see there are helpers e.g. `get_env_var_bool` and `get_env_var_float`, I wondered if it was worth adding `get_env_var_int` for consistency

**Review by frtibble — DISMISSED** [review]

**Comment by cecheta** [comment]
> LGTM! I assume search returns images in order of closest match?

Yep, this is from the docs:

> If no OrderBy is specified, the default sort order is descending by document match score.

File: code/backend/batch/utilities/helpers/env_helper.py (hunk 1 above)
Comment (cecheta) [reply code/backend/batch/utilities/helpers/env_helper.py]: Good point, added

**Review by adamdougal — APPROVED** [review]

**Pull request merged**